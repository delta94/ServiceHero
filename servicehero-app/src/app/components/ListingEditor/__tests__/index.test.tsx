import * as React from 'react';
import { render } from '@testing-library/react';
import { ListingEditor } from '..';
import { Provider } from 'react-redux';
import { configureAppStore } from 'store/configureStore';

describe('<ListingEditor  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <Provider store={configureAppStore()}>
        <ListingEditor />
      </Provider>,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
