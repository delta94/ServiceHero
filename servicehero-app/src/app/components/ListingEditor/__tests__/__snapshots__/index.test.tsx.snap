// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListingEditor  /> should match snapshot 1`] = `
.c1 {
  width: 400px;
  border-radius: 8px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  text-align: center;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  class="c0"
>
  <div
    class="c1"
  >
    <div
      class="ant-divider ant-divider-horizontal ant-divider-with-text ant-divider-with-text-center"
      role="separator"
    >
      <span
        class="ant-divider-inner-text"
      >
        <h4
          class="ant-typography"
        >
          Add new
           listing
        </h4>
      </span>
    </div>
    <form
      class="ant-form ant-form-horizontal ant-form-large"
      id="basic"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-select ant-select-lg ant-select-single ant-select-show-arrow"
              >
                <div
                  class="ant-select-selector"
                >
                  <span
                    class="ant-select-selection-search"
                  >
                    <input
                      aria-activedescendant="basic_type_list_0"
                      aria-autocomplete="list"
                      aria-controls="basic_type_list"
                      aria-haspopup="listbox"
                      aria-owns="basic_type_list"
                      autocomplete="off"
                      class="ant-select-selection-search-input"
                      id="basic_type"
                      readonly=""
                      role="combobox"
                      style="opacity: 0;"
                      type="search"
                      unselectable="on"
                      value=""
                    />
                  </span>
                  <span
                    class="ant-select-selection-placeholder"
                  >
                    Service type
                  </span>
                </div>
                <span
                  aria-hidden="true"
                  class="ant-select-arrow"
                  style="user-select: none;"
                  unselectable="on"
                >
                  <span
                    aria-label="down"
                    class="anticon anticon-down ant-select-suffix"
                    role="img"
                  >
                    <svg
                      aria-hidden="true"
                      data-icon="down"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      viewBox="64 64 896 896"
                      width="1em"
                    >
                      <path
                        d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                      />
                    </svg>
                  </span>
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <textarea
                class="ant-input ant-input-lg"
                id="basic_description"
                maxlength="140"
                placeholder="Description"
              />
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <input
                class="ant-input ant-input-lg"
                id="basic_hourlyRate"
                placeholder="Hourly rate â‚¬"
                type="number"
                value=""
              />
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-label"
        >
          <label
            class="ant-form-item-required"
            for="basic_availability"
            title="Availability"
          >
            Availability
          </label>
        </div>
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="ant-picker ant-picker-range ant-picker-large"
                style="float: right;"
              >
                <div
                  class="ant-picker-input ant-picker-input-active"
                >
                  <input
                    autocomplete="off"
                    id="basic_availability"
                    placeholder="Start time"
                    readonly=""
                    size="10"
                    value=""
                  />
                </div>
                <div
                  class="ant-picker-range-separator"
                >
                  <span
                    aria-label="to"
                    class="ant-picker-separator"
                  >
                    <span
                      aria-label="swap-right"
                      class="anticon anticon-swap-right"
                      role="img"
                    >
                      <svg
                        aria-hidden="true"
                        data-icon="swap-right"
                        fill="currentColor"
                        focusable="false"
                        height="1em"
                        viewBox="0 0 1024 1024"
                        width="1em"
                      >
                        <path
                          d="M873.1 596.2l-164-208A32 32 0 00684 376h-64.8c-6.7 0-10.4 7.7-6.3 13l144.3 183H152c-4.4 0-8 3.6-8 8v60c0 4.4 3.6 8 8 8h695.9c26.8 0 41.7-30.8 25.2-51.8z"
                        />
                      </svg>
                    </span>
                  </span>
                </div>
                <div
                  class="ant-picker-input"
                >
                  <input
                    autocomplete="off"
                    placeholder="End time"
                    readonly=""
                    size="10"
                    value=""
                  />
                </div>
                <div
                  class="ant-picker-active-bar"
                  style="left: 0px; width: 0px; position: absolute;"
                />
                <span
                  class="ant-picker-suffix"
                >
                  <span
                    aria-label="clock-circle"
                    class="anticon anticon-clock-circle"
                    role="img"
                  >
                    <svg
                      aria-hidden="true"
                      data-icon="clock-circle"
                      fill="currentColor"
                      focusable="false"
                      height="1em"
                      viewBox="64 64 896 896"
                      width="1em"
                    >
                      <path
                        d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
                      />
                      <path
                        d="M686.7 638.6L544.1 535.5V288c0-4.4-3.6-8-8-8H488c-4.4 0-8 3.6-8 8v275.4c0 2.6 1.2 5 3.3 6.5l165.4 120.6c3.6 2.6 8.6 1.8 11.2-1.7l28.6-39c2.6-3.7 1.8-8.7-1.8-11.2z"
                      />
                    </svg>
                  </span>
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <button
                class="ant-btn ant-btn-primary ant-btn-lg ant-btn-block"
                type="submit"
              >
                <span>
                  Save
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </form>
  </div>
</div>
`;
